docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/05 - Visits.md

```diff
@@ -0,0 +1,44 @@
+# Visits
+
+## Overview
+The **Visit** designates the instance in time where a candidate has visited or contacted a site and underwent any sort of data collection.
+
+**Visit** and **Timepoint** are terms used interchangeably in LORIS. **Visits** are defined in the `Visit_Windows` table of the database.
+
+## Adding Options
+
+### Front End
+ _not yet available_
+
+### SQL (Recommended)
+
+This table must be populated with visit labels - the Imaging Pipeline critically depends on this.  To populate with visit labels, run tools/populate_visit_windows.php - or manually insert study-specific information:
```

Comment:
I think this is a little misleading.. the populate_visit_windows was a one-time script to populate it based on the old config.xml settings. It can't pull the values out of nowhere, and most new people reading this shouldn't have to run this (it should have been in the release notes of whichever release included it.)

Reply:
I just realized it says to configure the config.xml below.. in that case, this should say that it requires the config.xml be set up (as below) before running the script (and probably be moved to after that part of the document)

Reply:
I agree with @driusan. Would be best to mention how to populate `config.xml` first and then ask them to run the script `tools/populate_visit_windows.php`.

Also, there should be back tick around `tools/populate_visit_windows.php`

Reply:
i will remove it completely I dont know why we would first populate config.xml and then not use it lol